keyboards/keyprez/unicorn/info.json

```
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "Unicorn",
+    "url": "",
+    "maintainer": "Keyprez",
+    "layouts": {
+        "LAYOUT_ISO": {
```

```suggestion
        "LAYOUT": {
```

`LAYOUT_iso` would also be accepted.